batcher/batcher.mligo

```diff
@@ -239,6 +240,8 @@ type batch_ordertypes = (nat,  ordertypes) map
 (* Associated user address to a given set of batches and ordertypes  *)
 type user_batch_ordertypes = (address, batch_ordertypes) big_map
 
+(* The count of unredeemed addresses associated with each batch. This is used to clear fully redeedmed batches *)
+type batch_holdings = (nat,nat) big_map
```

Comment:
Instead of adding yet another big map, have you considered adding a counter to each batch?

Reply:
I considered that at first.  In the end I wanted to be able to see the unredeemed counts for a given batch.  I was thinking it may be useful for the UI.   That said, it was a bit of a whim and there isn't a reason we couldn't do it that way.

Reply:
I'll branch off a second implementation for this with the counter on the batch.  If it looks good we can go with that instead